tests/ferret/core_test.go

```diff
@@ -111,7 +111,7 @@ func TestCore(t *testing.T) {
 			"bool-false": false,
 			"bool-true":  true,
 
-			"datetime":          time.Date(2021, 11, 1, 10, 18, 42, 123000000, time.UTC),
+			"datetime":          primitive.DateTime(1635761922123),
```

Comment:
I assume we use that type because that's what we get back? Then we should change the next few values too. And let's use `primitive.NewDateTimeFromTime` to keep value human-readable.

Reply:
Values have been changed.